packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -103,12 +103,12 @@ export class ConversationService {
 
   constructor(
     private readonly apiClient: APIClient,
-    private readonly cryptographyService: CryptographyService,
+    cryptographyService: CryptographyService,
     private readonly assetService: AssetService,
   ) {
     this.messageTimer = new MessageTimer();
     this.messageBuilder = new MessageBuilder(this.apiClient, this.assetService);
-    this.messageService = new MessageService(this.apiClient, this.cryptographyService);
+    this.messageService = new MessageService(this.apiClient, cryptographyService);
```

Comment:
Now, the `convesationService` doesn't need to use the `cryptoService` anymore, it just passes it on to the message service.